src/editor.rs

```diff
@@ -353,58 +344,75 @@ impl<'arena, Node: Ast<'arena> + 'arena> Editor<'arena, Node> {
     /// Insert new child as the first child of the selected node
     fn insert_child(&mut self, c: char) {
         let cursor = self.tree.cursor();
-        if cursor.is_insert_char(c) {
-            if let Some(node) = cursor.from_char(c) {
-                if let Err(e) = self.tree.insert_child(node) {
-                    log::error!("{}", e);
-                } else {
-                    log::debug!("Inserting with '{}'", c);
-                }
-            } else {
+        //short circuit if not an insertable char
```

Comment:
Small nitpick: Could we have one space and a capital letter at the start of comments `;)`, for style consistency?

Like: `// Short circuit ...`